src/Shared/HttpSys/RequestProcessing/PathNormalizer.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
We should probably do a follow up PR to share this file

Reply:
I can follow up if I don't forget (holidays coming soon that wipe my memory)

Reply:
https://github.com/dotnet/aspnetcore/pull/57178